test/integration/entity-spec.ts

```diff
@@ -78,13 +79,20 @@ function entityStateMatchesExpected(actual: EntityState, expected: EntityState):
     }
 }
 
-class MockContext {
+class MockContext implements IEntityFunctionContext {

```

Comment:
Can you remind me why the `IEntityFunctionContext` is needed now?

Reply:
To get the tests to properly understand type inference the context that is used in the tests now implements an actual context. Previously, the arguments to the test functions were `any` typed so a ducktyped class was usable.